javascript/index.js

```diff
@@ -0,0 +1,14 @@
+// email validation
+const formVal = document.getElementById('formV');
+const emailInput = document.getElementById('mail');
+
+formVal.addEventListener('submit', (event) => {
+  const validMail = /[A-Z]/g;
+
+  if (validMail.test(emailInput.value)) {
```

Comment:
- [ ] The email should validate just lowercase characters. All email address by convention uses lowercase characters so it's important to add validation for this.

- [ ] The students can check if the email is all lowercase by comparing if the email and it's lowercase converted version are the same, ie: `email === email.toLowerCase()`, utilizing regular expressions, or looping through each letter of the email string. Any implementation of this check should be considered valid as long as it works properly.